pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/documentdb/index.ts

```diff
@@ -5,10 +5,12 @@ import * as pulumi from "@pulumi/pulumi";
 import * as utilities from "../utilities";
 
 // Export members:
-export * from "./sqlResourceSqlContainer";
 
-// Import resources to register:
-import { SqlResourceSqlContainer } from "./sqlResourceSqlContainer";
+export { SqlResourceSqlContainerArgs } from "./sqlResourceSqlContainer";
+export type SqlResourceSqlContainer = import("./sqlResourceSqlContainer").SqlResourceSqlContainer;
+export const SqlResourceSqlContainer: typeof import("./sqlResourceSqlContainer").SqlResourceSqlContainer = null as any
```

Comment:
3. mock-reexport the value of the resource class, this is needed to type-check expressions such as new SqlResourceSqlContainer(); still this generates no require()